### PR TITLE
Feature/us 1093 - Refactoring and clean-up pre-demo

### PR DESCRIPTION
**Pull request by jmiller-nexient**

### CODE REVIEW

File: app/components/Header/index.less [hunk 1]
```diff
@@ -1,67 +1,86 @@
 @import '../../styles/themes/index.less';
 
-@desktop: ~"(min-width: 451px)";
-@mobile: ~"(max-width: 450px)";
+@desktop: ~'(min-width: 451px)';
+@mobile: ~'(max-width: 450px)';
 
 .dte-wismo-header-background {
```
Comment (anoveskey) [inline app/components/Header/index.less:6]: why aren't you using the `dte-prefix` value as part of the class name?

File: app/components/Header/index.less [hunk 2]
```diff
@@ -1,67 +1,86 @@
...
-    position: relative;
+  background-color: @dte-background-brand-blue;
+  color: @dte-content-primary-inverse;
+  height: 96px;
+  top: 0px;
+  width: 100%;
+  position: relative;
 }
 
 .dte-wismo-header-link {
```
Comment (anoveskey) [inline app/components/Header/index.less:15]: same here.

File: app/components/Header/index.less (hunk 1 above)
Comment (jmiller-nexient) [reply app/components/Header/index.less:6]: We will be addressing these things this afternoon.  We merged both Jeremy's and Alex's PRs without a real review this morning.  We will perform that review post-merge next and these things will be captured then.

File: app/components/Header/index.less (hunk 2 above)
Comment (jmiller-nexient) [reply app/components/Header/index.less:15]: We will be addressing these things this afternoon.  We merged both Jeremy's and Alex's PRs without a real review this morning.  We will perform that review post-merge next and these things will be captured then.

File: app/components/Header/tests/index.test.tsx [hunk 3]
```diff
@@ -1,11 +1,13 @@
-import React, { ReactElement } from 'react';
+import React from 'react';
 import renderer from 'react-test-renderer';
 
+import TestRendererJsonType from 'types/TestRendererJsonType';
+
 import Header from '..';
```
Comment (anoveskey) [inline app/components/Header/tests/index.test.tsx:6]: relative path?

**Pull request merged**